gemstone/src/swapper/thorchain/model.rs

```diff
@@ -0,0 +1,75 @@
+use primitives::Chain;
+use serde::{Deserialize, Serialize};
+
+pub enum ThorChainAsset {
+    Doge,
+    Thorchain,
+    Ethereum,
+    Cosmos,
+    Bitcoin,
+    Litecoin,
+    SmartChain,
+    AvalancheC,
+}
+
+// https://dev.thorchain.org/concepts/memo-length-reduction.html
+impl ThorChainAsset {
+    pub fn short_name(&self) -> &str {
+        match self {
+            ThorChainAsset::Doge => "d",               // DOGE.DOGE
+            ThorChainAsset::Thorchain => "r",          // THOR.RUNE
+            ThorChainAsset::Ethereum => "e",           // "ETH.ETH"
+            ThorChainAsset::Cosmos => "g",             // GAIA.ATOM
+            ThorChainAsset::Bitcoin => "b",            // BTC.BTC
+            ThorChainAsset::Litecoin => "l",           // LTC.LTC
+            ThorChainAsset::SmartChain => "s",         // BSC.BNB
+            ThorChainAsset::AvalancheC => "AVAX.AVAX", // AVAX.AVAX. not sure
```

Comment:
replace with todo()!? this doesn't look short name

Reply:
fixed